backend/app/anti_recommenders/open_ai/regular_open_ai_anti_recommender.py

```diff
@@ -0,0 +1,85 @@
+import logging
+from typing import Tuple
+from langchain_openai import OpenAI
+from langchain.prompts import PromptTemplate
+from langchain.schema.runnable import RunnablePassthrough
+from langchain.schema import StrOutputParser
+
+from .open_ai_anti_recommender import OpenAIAntiRecommender
+
+
+class RegularOpenAIAntiRecommender(OpenAIAntiRecommender):
+    """A subclass of the OpenAI Anti-Recommender"""
+    
+    def build_model(self):
+        """Build and return an LLM"""
+        
+        try:
+            model = OpenAI()
+            prompt = PromptTemplate.from_template(super().get_template())
+
+            chain = (
+                {"question": RunnablePassthrough()}
+                    | prompt
+                    | model
+                    | StrOutputParser()
+            )
+
+            return chain
+        
+        except Exception:
```

Comment:
Ruff will tell you not to catch-all. Generally good practice.

Reply:
I have improved the Exception handling here and in other parts of the project.